lite/examples/bert_qa/android/README.md

```diff
@@ -17,7 +17,9 @@ These instructions walk you through running the demo on an Android device.
     [Android Studio](https://developer.android.com/studio/index.html), following
     the instructions on the website.
 
-*   Android Studio 3.2 or later.
+*   Android Studio 3.2 or later.  
```

Comment:
Please remove the trailing whitespace in these lines,  thanks!

Reply:
Done.